packages/fxa-auth-server/lib/senders/emails/templates/cadReminder/index.scss

```diff
@@ -0,0 +1,30 @@
+@use '../../global.scss';
+
+@mixin text-xl {
+  font-size: 20px !important;
+  line-height: 28px !important;
+}
+@mixin text-sm {
+  font-size: 14px !important;
+  line-height: 20px !important;
+}
+
+.header-text div {
+  @include text-xl;
+  font-family: global.$font-sans;
+  text-align: center !important;
+}
+
+span {
+  color: #0a84ff;
```

Comment:
Hmm, since we're using that color here too, let's pull that `$blue-400` into globals and then use it in both files.